### PR TITLE
VIH-9586 Hide save and edit buttons

### PR DESCRIPTION
**Pull request by abbasiTahir**
changed the html and typscript files to check if VHo has work hours

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/VIH-9586

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by marcogagliardi — APPROVED** [review]
👌🏽

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts [hunk 1]
```diff
@@ -480,4 +480,7 @@ export class VhoWorkHoursNonAvailabilityTableComponent implements OnInit, CanDea
     resetStartDateAndEndDate() {
         this.filterForm.setValue({ startDate: null, endDate: null });
     }
+    get checkVhoHasWorkHours(): boolean {
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts:483]: More of a naming style here but not a deal breaker. If returning a bool, prefix the method with 'if' or 'has'. turns this into a question to make it more readable and reflective of the behaviour. for example. `hasVHOGotWorkingHours`

Comment (abbasiTahir) [reply AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-non-availability-table/vho-work-hours-non-availability-table.component.ts:483]: Thank you Shead .

File: AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.ts [hunk 2]
```diff
@@ -157,4 +157,8 @@ export class VhoWorkHoursTableComponent implements CanDeactiveComponent {
     registerUnsavedChanges() {
         this.videoHearingsService.setVhoNonAvailabiltiesHaveChanged(true);
     }
+
+    get checkVhoHasWorkHours(): boolean {
```
Comment (shaed-parkar) [inline AdminWebsite/AdminWebsite/ClientApp/src/app/work-allocation/edit-work-hours/vho-work-hours-table/vho-work-hours-table.component.ts:161]: A recommended naming style for methods that return a bool is to prefix with with 'has' or 'is' to better reflect the behaviour of the method as a question. For example, `hasVhoGotWorkingHours` (assuming I got the context of the method right there)

**Pull request merged**